pygmt/base_plotting.py

```diff
@@ -181,6 +181,8 @@ def grdcontour(self, grid, **kwargs):
             Do not draw contours with less than `cut` number of points.
         S : string or int
             Resample smoothing factor.
+        l : str
```

Comment:
grdcontour doesn't support `-l` option now. Currently only `plot` supports auto-label.

Reply:
Resolving in https://github.com/GenericMappingTools/pygmt/pull/357.